source/mongostat.txt

```diff
@@ -401,15 +401,16 @@ Options
 
 .. option:: -o=<field list>
 
-   When specified, :program:`mongostat` includes **only** the specified fields
-   in the :program:`mongostat` output.
+   When specified, :program:`mongostat` includes **only** the specified
```

Comment:
The initial group of changes, from lines 404 - 465, include a lot of line-length fixes.

The content updates for that section are on lines:
411-13
460-62